deepchem/models/tensorgraph/models/scscore.py

```diff
@@ -0,0 +1,39 @@
+import deepchem as dc
+import tensorflow as tensorflow
+from deepchem.models.tensorgraph.layers import Layer, Input, Dense, Hingeloss, Label, Feature
+from deepchem.models.tensorgraph.layers import ReduceMean, InputFifoQueue, ReLU
+
+
+class Scscore(TensorGraph):
+
+  tg = dc.models.TensorGraph(
+      tensorboard=True, model_dir='/tmp/scscore', use_queue=True)
+
+  reactant_features = Feature()
+  product_features = Feature()
+
+  dense_reactant_1 = Dense(
+      out_channels=300, in_layers=[reactant_features], activation_fn=tf.nn.relu)
+  dense_product_1 = dense_reactant_1.shared(in_layers=[product_features])
+  dense_reactant_2 = Dense(
+      out_channels=300, in_layers=[dense_reactant_1], activation_fn=tf.nn.relu)
+  dense_product_2 = dense_reactant_2.shared(in_layers=[dense_product_1])
+  dense_reactant_3 = Dense(
+      out_channels=300, in_layers=[dense_reactant_2], activation_fn=tf.nn.relu)
+  dense_product_3 = dense_reactant_3.shared(in_layers=[dense_product_2])
+  dense_reactant_4 = Dense(
+      out_channels=300, in_layers=[dense_reactant_3], activation_fn=tf.nn.relu)
+  dense_product_4 = dense_reactant_4.shared(in_layers=[dense_product_3])
+  dense_reactant_5 = Dense(
+      out_channels=300, in_layers=[dense_reactant_4], activation_fn=tf.nn.relu)
+  dense_product_5 = dense_reactant_5.shared(in_layers=[dense_product_4])
+
+  output_dense = Add(in_layers=[dense_reactant_5, dense_product_5])
```

Comment:
I don't think this is what you want to do.

Reply:
In the scscore paper the author says that they use 2 models to calculate the 2 scores seperately for reactant and product and then the difference between their scores are evaluated.
So is my mistake that I dont have to add the layers and continue with the 2 layers for reactant and product and the evaluate the score for them and then calculate the difference from the scores?